docs/src/library/function_index.md

```diff
@@ -1,5 +1,5 @@
 ### [Index](@id main-index)
 
 ```@index
-Pages = ["api.md", "function_index.md"]
+Pages = ["lorentz_index.md", "function_index.md"]
```

Comment:
I think just the empty `@index` is enough for a list of all the functions and types.
```suggestion
```